types/wait-for-element-to-be-removed.d.ts

```diff
@@ -0,0 +1,8 @@
+export function waitForElementToBeRemoved<T>(
```

Comment:
Fyi, I made a PR to DefinitelyTyped to add the `interval` option here.
It got merged today, so I believe we should also add it here.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44006/files

Reply:
actually since options is simply passed to waitFor, it should just use the same type, so we don't have to redefine it in multiple places, I'll take care of it

Reply:
That's a good idea 👍